lib/throttle.js

```diff
@@ -25,6 +25,21 @@ module.exports = (api) => {
     };
   };
 
+  api.metasync.debounce = (
+    timeout,
+    fn,
+    args
+  ) => {
+    let timer = null;
+    return () => {
+      clearTimeout(timer);
+      timer = setTimeout(
+        () => (args !== undefined ? fn(...args) : fn()),
+        timeout
```

Comment:
Lets consider the case when user has input field and `keyup` handler on it. If we pass an debounced function as a handler, with for example 500 ms timeout, that while user press a keys with friqency more than 500 and then stops. Debounced function will be called one time.

That is my consideration of its proper way of the work.